responses.go

```diff
@@ -222,9 +239,18 @@ type AccountResp struct {
 	CPULimit               AccountResourceLimit `json:"cpu_limit"`
 	Permissions            []Permission         `json:"permissions"`
 	TotalResources         TotalResources       `json:"total_resources"`
-	SelfDelegatedBandwidth DelegatedBandwidth   `json:"self_delegated_bandwidth"`
+	SelfDelegatedBandwidth *DelegatedBandwidth  `json:"self_delegated_bandwidth"`
```

Comment:
And this one? So that it can be `null`?

Reply:
https://github.com/psy2848048/eos-go/blob/b489eeb186b3aa46096c1deff3f75ed651d6ebf8/testdata/mock_server/chain_get_account.json#L238

Reply:
Rollbacked as an actual object and manipulated the expected result

https://github.com/eoscanada/eos-go/pull/191/commits/ca5b8435b0ca0323c521abc3d07eb6985015a0f6